openshift4/templates/filesystem-provider.dc.yaml

```diff
@@ -2,6 +2,12 @@ kind: Template
 apiVersion: v1
 metadata:
   name: mds-filesystem-provider-dc
+parameters:
+  - name: APPNAME
+    displayName: AppName
+    description: Application name for grouping pods in Openshift
+    value: Core
+    required: true
 objects:
```

Comment:
@jon-funk I'm not sure why these are in our PR - I'm thinking I didn't resolve the merge conflicts correctly. Can you let me know what this and the redis.dc.yaml files should be.

Reply:
@yasserhu do you mind taking a look at these two yaml files and advise. I think this is the result of merge conflicts as we did not add this code.